### DISCUSSION TITLE
View all open pull requests

### DISCUSSION THREAD

**Discussion by matthoward** [community/community#4165] [Mobile] [answered]
I'd like to be able to see all open PRs for my repos, not just the ones where I'm the creator, assignee, mentioned or a reviewer. 

![image](https://user-images.githubusercontent.com/2996572/121774001-a7056100-cb4d-11eb-8312-fca8fe1eae04.png)

It's also a little confusing since the "Created" filter is actually always active even though it shows as greyed out. If I change the dropdown to "Mentioned" it turns blue showing the filter is active. Changing to "Created" turns grey implying there isn't any filter being applied. This is actually what I'd like (don't filter for "my" PRs) but there isn't actually any way to turn off that filter.

**Comment by alcere** [comment, marked as answer]
Hi! Thanks for your feedback! We've shared your comments with the design team. We are currently discussing how we can improve the filtering experience, and these opinions will be very useful!

**Reply by kronn** [reply]
While this answer is a nice first step, nothing on this changed since. I want to get an overview over all PRs in an organization or just a repo. Our team mostly works by picking a PR not assigned. For this, an overview over all PRs would be helpful.

**Comment by Jose1370** [comment]
G[]()
